### PR TITLE
Add examples for Accordion

### PR DESCRIPTION
**Pull request by agatac**
Issue: #36 

Added an example for nested accordions so far.

### CODE REVIEW

**Comment by agatac** [comment]
There were a couple issues I came across:
1. It seems that `fluid` prop isn't added to the list of classes in the final html. I can add `class="fluid"` instead and the width is correct.
2. I couldn't find sui radio checkboxes to use in Accordion>Examples>Accordion Menu ([from original docs](https://semantic-ui.com/modules/accordion.html#accordion-menu)). Shouldn't be an issue unless you want to be 100% the same as the original docs.

**Comment by mariolamacchia** [comment]
You can skip examples for things that aren't implemented for now :)

**Comment by agatac** [comment]
Just making sure I'm not missing anything :)

Alright, I think it's ready to merge.

File: docs/examples/modules/AccordionExample/index.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import Accordion from './Accordion.example';
 import AccordionStyled from './AccordionStyled.example';
-import AccordionFluid from './AccordionFluid.example';
```
Comment (mariolamacchia) [inline docs/examples/modules/AccordionExample/index.js]: I think we can remove the file too at this point

Comment (agatac) [reply docs/examples/modules/AccordionExample/index.js]: Done

**Pull request merged**